models/articles.model.js

```diff
@@ -8,4 +8,52 @@ exports.fetchArticleById = (article_id) => {
           }
           return rows[0];
         });
-}
\ No newline at end of file
+}
+
+exports.fetchArticles = (author, topic, sort_by = "created_at", order = "desc") => {
```

Comment:
Great handling of this method - see above comment re the additional code - but the fundamental aspects required here look good, well done!